stac_fastapi/sqlalchemy/alembic/versions/5909bd10f2e6_change_item_geometry_column_type.py

```diff
@@ -0,0 +1,35 @@
+"""change item geometry column type
+
+Revision ID: 5909bd10f2e6
+Revises: 821aa04011e8
+Create Date: 2021-11-23 10:14:17.974565
+
+"""
+import sqlalchemy as sa
```

Comment:
Checked linting, just need to remove this line for flake8 to pass

Reply:
Thanks @lossyrob - just did that and also rebased from current master to avoid conflicts.